closed/Intel/code/README.md

```diff
@@ -46,7 +46,7 @@ cd resnet50/pytorch-cpu/
 cd dlrm-99.9/pytorch-cpu/
 ```
 ```
-../../run_docker.sh 038cf950d509  /workspace/RUN.sh  /opt/workdir/code/dlrm-99.9/pytorch-cpu aws_dlrm
+../../run_docker.sh 038cf950d509  /workspace/run_offline.sh  /opt/workdir/code/dlrm-99.9/pytorch-cpu aws_dlrm
```

Comment:
possible also add information for other tests like offline_accuracy, server, and server_accuracy ?
You could just add a note mentioned how to apply other run script and what scripts are available.